pkg/drivers/drivers.go

```diff
@@ -136,3 +139,32 @@ func fixPermissions(path string) error {
 	}
 	return nil
 }
+
+// ValidatePermissions validates minikube is run by the recommended user (privileged or regular)
+func ValidatePermissions(driverName string) error {
```

Comment:
I don't think this function should be in this package. This is related to what @afbjorklund was taking in the issue https://github.com/kubernetes/minikube/issues/4909. Anything under `pkg/drivers` shouldn't be using minikube code directly, because the idea is that those drivers will be moved upstream. In this case, we have translations here. Drivers code related to minikube should be in `pkg/minikube/drivers`.

wdyt?

Reply:
In one of the review comments above, Thomas mentioned that it would be better if we could place this function at this path.

Reply:
Gotcha! No strong feelings about it. Although I think pkg/drivers should be drivers only, not using minikube code.

@tstromberg thoughts?

Reply:
SGTM. I think this should go in `pkg/minikube/drivers/drivers.go` then?